src/main/java/dinkplugin/SettingsManager.java

```diff
@@ -181,9 +188,50 @@ void onConfigChanged(ConfigChanged event) {
         }
     }
 
+    void onGameState(GameStateChanged event) {
+        if (event.getGameState() != GameState.LOGGED_IN) {
+            justLoggedIn.set(false);
+            return;
+        } else {
+            justLoggedIn.set(true);
+        }
```

Comment:
Does this improve readability?

```suggestion
        boolean loggedIn = event.getGameState() == GameState.LOGGED_IN;
        justLoggedIn.set(loggedIn);
        if (!loggedIn) return;
```

Reply:
I think this is readable enough already